src/pages/MainPage/MainPage.tsx

```diff
@@ -128,6 +150,16 @@ export const MainPage: React.FC = () => {
             {boxes.map((box, index) => (
               <Box key={`${box}-${index}`} number={box} index={index} />
             ))}
+            <div
+              className={`main-page__play__boxes__grid__prisoner ${
+                gameRunning
+                  ? "main-page__play__boxes__grid__prisoner--active"
+                  : ""
+              }`}
+              id="floating-prisoner"
```

Comment:
use `ref` over `id` in these cases, where you are using the same thing programmatically in code

Reply:
done